ext/doc/Cassandra.php

```diff
@@ -322,7 +322,12 @@
     /**
      * Current version of the extension.
      */
-    const VERSION = '1.0.0-beta';
+    const VERSION = '1.1';
+
+    /**
+     * Version of the cpp-driver the extension is compiled against.
+     */
+    const CPP_DRIVER_VERSION = '2.2.2';
```

Comment:
This might not necessarily be true in the event a different version of the cpp-driver is used. Can we use the [defines in `cassandra.h`](https://github.com/datastax/cpp-driver/blob/master/include/cassandra.h#L53-L56) instead to be dynamic?